drone_circuit_assets/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(jderobot_assets)
+
+## ROS INSTALL ##
+
+find_package(catkin REQUIRED COMPONENTS)
```

Comment:
`drone_assets` should be required since drone models are needed

Reply:
Resolved